zc_common/remote_resource/relations.py

```diff
@@ -36,8 +37,15 @@ def get_links(self, obj=None, lookup_field='pk'):
         self_link = self.get_url('self', self.self_link_view_name, self_kwargs, request)
 
         # Construct the related link using the passed related_resource_path
-        #   self.source is the field name; getattr(obj, self.source) returns the RemoteResource object
-        related_path = self.related_resource_path.format(pk=getattr(obj, self.source).id)
+        # self.source is the field name; getattr(obj, self.source) returns the
+        # RemoteResource object or RelatedManager in the case of a to-many relationship.
+        related_obj = getattr(obj, self.source)
+        if isinstance(related_obj, BaseManager):
+            list_of_ids = related_obj.values_list('id', flat=True)
+            query_parameters = unquote(urlencode({'ids[]': list_of_ids}, doseq=True))
+            related_path = self.related_resource_path.format(pk=query_parameters)
+        else:
+            related_path = self.related_resource_path.format(pk=related_obj.id)
```

Comment:
Agreed, I'll update the `values_list()` method call as well to get a list of pks instead of assuming ids.